templates/docs/examples/patterns/navigation/search-dark.html

```diff
@@ -0,0 +1,109 @@
+{% extends "_layouts/examples.html" %}
+
+{% block title %}Navigation / Search dark{% endblock %}
+
+{% block standalone_css %}patterns_navigation{% endblock %}
+
+{% block content %}
+
+<header id="navigation" class="p-navigation is-dark">
+  <div class="p-navigation__row">
+    <div class="p-navigation__banner">
+      <div class="p-navigation__logo">
+        <a class="p-navigation__item" href="#">
+          <img class="p-navigation__image" src="https://assets.ubuntu.com/v1/3c7954dd-logo-canonical-white.svg" alt="" width="95" />
+        </a>
+      </div>
+      <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+      <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+    </div>
+    <nav class="p-navigation__nav" aria-label="Example main navigation">
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item is-selected">
+          <a class="p-navigation__link" href="#">Products</a>
+        </li>
+        <li class="p-navigation__item">
+          <a class="p-navigation__link" href="#">Services</a>
+        </li>
+        <li class="p-navigation__item">
+          <a class="p-navigation__link" href="#">Partners</a>
+        </li>
+        <li class="p-navigation__item">
+          <a class="p-navigation__link" href="#">About</a>
+        </li>
+			</ul>
+			<ul class="p-navigation__items">
+				<li class="p-navigation__item" id="link-4">
+					<a href="#" class="js-search-button p-navigation__link">
+						<span class="u-hide u-show--large">Search</span> <i class="p-icon--search is-light"></i>
+					</a>
+        </li>
+			</ul> 
+      <div class="p-navigation__search u-show--small u-hide" style="z-index: 39;">
+        <form action="/search" class="p-search-box" id="ubuntu-global-search-form">
+          <input type="search" class="p-search-box__input" name="q" placeholder="Search our sites" required="" aria-label="Search our sites">
+          <button type="reset" class="p-search-box__reset"><i class="p-icon--close"></i></button>
+          <button type="submit" class="p-search-box__button"><i class="p-icon--search">Search</i></button>
+        </form>
+      </div>
+    </nav>
+  </div>
+</header>
+
+<script>
+	function initSearch() {
```

Comment:
It's time I guess to move it to separate file.

I think we can put it alongside these examples in same folder, similar to [_script.js](https://github.com/canonical-web-and-design/vanilla-framework/blob/master/templates/docs/examples/patterns/navigation/_script.js), maybe call it `_script-search.js` or something.

You can include it in a template like this:
https://github.com/canonical-web-and-design/vanilla-framework/blob/60397cbee2e67d95c0b7d3fb46de9cea9d94f695/templates/docs/examples/patterns/navigation/dropdown-dark.html#L86

Reply:
Thanks @bartaz for reviewing. I created the `_script-search.js` file in the same folder. :)